DESCRIPTION

```diff
@@ -32,7 +32,9 @@ Suggests:
     covr,
     knitr,
     rmarkdown,
-    bit64
+    bit64,
+    readr,
```

Comment:
I don't think we should depend on `readr` here, can we use `read.csv`? There are a lot of dependencies in `readr`, so I much rather avoid the build times and breaking changes if possible. `fs` aand `bit64` make sense, but I'd really try to avoid `readr`.